frontend/src/utils/index.js

```diff
@@ -726,6 +726,7 @@ export function k8sVersionExpirationForShoot (shootK8sVersion, shootCloudProfile
   }
   return {
     expirationDate: version.expirationDate,
+    expirationDateString: getDateFormatted(version.expirationDate),
```

Comment:
Why do we add a formatted date String here and not do the formatted in the component template via computed property or method?

Reply:
Yes, this would be possible. The method is in utils to make testing easier. It is used in a computed property anyway, so we could map and enrich the result there. If you want, I can change it but I thought it would be cleaner to have it all in one place.